server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/DataTransferService.java

```diff
@@ -229,8 +229,8 @@ public DataTransferTaskContext create(@NonNull String bucket,
     public UploadFileResult getMetaInfo(@NonNull String fileName) throws IOException {
         File uploadFile = fileManager.findByName(TaskType.IMPORT, LocalFileManager.UPLOAD_BUCKET, fileName).orElseThrow(
                 () -> new FileNotFoundException("File not found"));
-        if (!uploadFile.exists() || !uploadFile.isFile()) {
-            throw new IllegalArgumentException("Target is not a file or does not exist, " + fileName);
+        if (!uploadFile.exists()) {
```

Comment:
we may add a parameter for distinguish import by file or directory

Reply:
added param:  fileType